.travis.yml

```diff
@@ -17,9 +15,15 @@ matrix:
     - php: 7.1
       env: PHPCS=1
 
+    - php: 5.6
+      env: PREFER_LOWEST=1
+
 install:
   - composer install --prefer-dist --no-interaction
 
+before_script:
+  - if [[ $PREFER_LOWEST == 1 ]]; then composer update --prefer-dist --no-interaction --prefer-lowest --prefer-stable ; fi
```

Comment:
Please use the linked Source of if else. This still runs 3.6.2. here now.

Reply:
the install runs again. check the whole output.

Reply:
@dereuromark  yeah 
added a commit